LICENSE

```diff
@@ -3,4 +3,6 @@ Bevy is dual-licensed under either
 * MIT License (docs/LICENSE-MIT or http://opensource.org/licenses/MIT)
 * Apache License, Version 2.0 (docs/LICENSE-APACHE or http://www.apache.org/licenses/LICENSE-2.0)
 
-at your option.
\ No newline at end of file
+at your option.
+
+FiraMono is licensed under the SIL Open Font License, Version 1.1 (assets/fonts/FiraMono-LICENSE)
```

Comment:
Maybe we can make something like a thirdparty licence file instead of adding this in Bevy's Licence?
The new file could then be linked in the readme (or the examples readme).

Reply:
or perhaps we can generalise this to be something to the effect of:
"Some asset files may be licensed under different terms where noted"

Reply:
Assets are not packaged with Bevy, but this license is. I think Licenses for assets should be elsewhere.

There are some in https://github.com/bevyengine/bevy/blob/main/CREDITS.md or in https://github.com/bevyengine/bevy/blob/main/assets/models/AlienCake/README.md but those looks like attributions for CC0 licenses

Reply:
On my own projects, I've been storing a .csv at the root of the `assets` folder that tracks licenses and attribution.

Reply:
Moved it to CREDITS.md.